pkg/cmd/pr/shared/commentable.go

```diff
@@ -50,6 +52,10 @@ func CommentablePreRun(cmd *cobra.Command, opts *CommentableOptions) error {
 		opts.InputType = InputTypeInline
 		inputFlags++
 	}
+	if cmd.Flags().Changed("body-file") {
+		opts.InputType = InputTypeFile
```

Comment:
`InputTypeFile` isn't actually used from anywhere. I do not think it's useful to declare it. We can just piggyback on InputTypeInline here (which is also not used, but at least it distinguishes from other types).